### PR TITLE
[mod_tags_popular]XML code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_tags_popular
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

File: modules/mod_tags_popular/mod_tags_popular.xml [hunk 1]
```diff
@@ -25,128 +25,133 @@
...
-					default="_:default"
 					label="JFIELD_ALT_LAYOUT_LABEL"
-					description="JFIELD_ALT_MODULE_LAYOUT_DESC" />
+					description="JFIELD_ALT_MODULE_LAYOUT_DESC"
+					default="_:default"
+				/>
+
 				<field
 					name="moduleclass_sfx"
 					type="textarea" rows="3"
```
Comment (andrepereiradasilva) [inline modules/mod_tags_popular/mod_tags_popular.xml]: shouldn't rows be moved down?

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on ce3a3736ecb4c54e3845a22db008a2189425d787<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10594">issues.joomla.org/joomla-cms/10594</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on ce3a3736ecb4c54e3845a22db008a2189425d787<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10594">issues.joomla.org/joomla-cms/10594</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10594">issues.joomla.org/joomla-cms/10594</a>.</sub>

**Pull request closed without merging**